src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerConfigureOptions.cs

```diff
@@ -0,0 +1,96 @@
+    /// </remarks>
+    [UnsupportedOSPlatform("browser")]
+    internal sealed class ConsoleLoggerConfigureOptions : IConfigureOptions<ConsoleLoggerOptions>
+    {
+        private readonly IConfiguration _configuration;
+
+        public ConsoleLoggerConfigureOptions(ILoggerProviderConfiguration<ConsoleLoggerProvider> providerConfiguration)
+        {
+            _configuration = providerConfiguration.Configuration;
+        }
+
+        public void Configure(ConsoleLoggerOptions options)
+        {
+            if (_configuration["DisableColors"] is string disableColors)
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                options.DisableColors = bool.Parse(disableColors);
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+
+            if (_configuration["Format"] is string format)
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                options.Format = ParseEnum<ConsoleLoggerFormat>(format);
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+
+            if (_configuration["FormatterName"] is string formatterName)
+            {
+                options.FormatterName = formatterName;
+            }
+
+            if (_configuration["IncludeScopes"] is string includeScopes)
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                options.IncludeScopes = bool.Parse(includeScopes);
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+
+            if (_configuration["LogToStandardErrorThreshold"] is string logToStandardErrorThreshold)
+            {
+                options.LogToStandardErrorThreshold = ParseEnum<LogLevel>(logToStandardErrorThreshold);
+            }
+
+            if (_configuration["MaxQueueLength"] is string maxQueueLength)
+            {
+                options.MaxQueueLength = int.Parse(maxQueueLength, NumberStyles.Integer, NumberFormatInfo.InvariantInfo);
+            }
+
+            if (_configuration["QueueFullMode"] is string queueFullMode)
+            {
+                options.QueueFullMode = ParseEnum<ConsoleLoggerQueueFullMode>(queueFullMode);
+            }
+
+            if (_configuration["TimestampFormat"] is string timestampFormat)
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                options.TimestampFormat = timestampFormat;
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+
+            if (_configuration["UseUtcTimestamp"] is string useUtcTimestamp)
+            {
+#pragma warning disable CS0618 // Type or member is obsolete
+                options.UseUtcTimestamp = bool.Parse(useUtcTimestamp);
+#pragma warning restore CS0618 // Type or member is obsolete
+            }
+        }
+
+        public static T ParseEnum<T>(string value) where T : struct =>
+#if NETSTANDARD || NETFRAMEWORK
```

Comment:
```suggestion
#if NETFRAMEWORK || NETSTANDARD2_0
```

Reply:
Isn't `NETFRAMEWORK` will included with  `NETSTANDARD2_0`? or we have check for both?

Reply:
You need to check for both. We build for the following TFMs:

https://github.com/dotnet/runtime/blob/8ad7113ac3e27453d3b9664181c0a5297323ae65/src/libraries/Microsoft.Extensions.Logging.Console/src/Microsoft.Extensions.Logging.Console.csproj#L4

Reply:
Yes, I don't know if the SDK is smart enough in defining these. I mean if defining `NETFRAMEWORK` is covering `NETSTANDARD2_0`. But anyway checking both never hurt :-)